automation/terraform/deployments/testworld-logging/output.tf

```diff
@@ -0,0 +1,7 @@
+output "docker_vm_ip" {
+  value = google_compute_instance.default.network_interface.0.access_config.0.nat_ip
+}
+
+# output "cloud_postgres_ip" {
```

Comment:
I understand uncommenting this will grant postgres instance an external ip, right?